codegen.ts

```diff
@@ -0,0 +1,23 @@
+import { CodegenConfig } from '@graphql-codegen/cli';
+import { WATSON_URL } from './src/lib/constants';
+
+const config: CodegenConfig = {
+    schema: [
+        {
+            ["https://watson-v2-staging.onrender.com/gql"]: {},
```

Comment:
@TosinJs this shouldn't be hardcoded, no? We don't want to hardcode a staging URL in the library permanently.